vibe_code_apps_22/scripts/inventory_greybox_sensors.py

```
@@ -55,39 +67,58 @@ def _columns_from_parquet(p: Path) -> set[str]:
             continue
         sources.append(str(p))
         if p.suffix == ".parquet":
             found_cols |= _columns_from_parquet(p)
-        else:
+        elif p.suffix == ".csv":
             try:
                 import pandas as pd
 
                 found_cols |= set(map(str, pd.read_csv(p, nrows=2).columns))
             except Exception:
                 pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)